library/src/main/kotlin/com/chuckerteam/chucker/internal/data/repository/HttpTransactionDatabaseRepository.kt

```diff
@@ -42,4 +42,13 @@ internal class HttpTransactionDatabaseRepository(private val database: ChuckerDa
     }
 
     override suspend fun getAllTransactions(): List<HttpTransaction> = transactionDao.getAll()
+
+    override fun getTransactions(
```

Comment:
```suggestion
    override fun getTransactionsInTimeRange(
```

or somethign more meaningful